compiler-core/src/language_server/engine.rs

```diff
@@ -368,6 +368,9 @@ where
 
         // Module types
         for (name, type_) in &module.ast.type_info.types {
+            if type_.internal {
+                continue;
+            }
```

Comment:
Are these lines of code excluding `@internal` types from the same file from completions? I don't think we want that.

Reply:
I think you're right! I will look into it
Thanks for pointing it out!!

Reply:
I checked and you're perfectly right! Thank you so much for the help 💜